rust/syntect-plugin/examples/make_manifest.rs

```diff
@@ -65,6 +66,16 @@ fn main() -> Result<(), io::Error> {
     let toml_str = toml::to_string(&mani).unwrap();
     let file_path = Path::new(OUT_FILE_NAME);
     let mut f = File::create(file_path)?;
+
+    // Create dump file
+    match dump_to_file(&syntax_set, "syntaxes.packfile") {
+        Ok(_) => (),
+        Err(err) => {
+            // Error Handling
+            eprintln!("{}", err);
```

Comment:
we should also just exit with an error code, if this happens we have a problem that needs attention. I think `panic!` might even be appropriate, here?

Reply:
Done.